RoeiVerenigingWPF/Pages/AddReservation.xaml.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+using DataAccessLibary;
+using RoeiVerenigingLibary;
+using RoeiVerenigingLibary.Exceptions;
+
+namespace RoeiVerenigingWPF.Frames
+{
+    /// <summary>
+    /// Interaction logic for AddReservation.xaml
+    /// </summary>
+    /// 
+    public partial class AddReservation
+    {
+        public DateTime? SelectedDate { get; set; }
+        private DateTime? StartTime { get; set; }
+        private DateTime? EndTime
+        {
+            get;
+            set;
+        }
+//placeholder for logged in member
+        private Member _member = new Member(30,  "rick", "wil", "roeien", "rick@windesheim.be", new List<string>());
```

Comment:
nu komt de reservering altijd op rick zijn naam te staan terwijl dit de ingelogde gebruiker moet zijn

Reply:
Staat al in master. Is in account aanmaken al toegepast